### PR TITLE
Storage

### PR DESCRIPTION
**Pull request by carriewright11**
- starting to add information about what people should look for in a repository
- also adding a bit more about how long people should retain data as I found more info about that

### CODE REVIEW

**Comment by carriewright11** [comment]
huh different tables: https://www.nlm.nih.gov/NIHbmic/nih_data_sharing_repositories.html want to check those out and compare with the link I have

**Comment by carriewright11** [comment]
> https://www.nlm.nih.gov/NIHbmic/nih_data_sharing_repositories.html

so I think this resource might have different info: https://www.nlm.nih.gov/NIHbmic/other_data_resources.html but I think this one mostly overlaps https://www.nlm.nih.gov/NIHbmic/domain_specific_repositories.html

File: 02-data_management_plan.Rmd [hunk 1]
```diff
@@ -16,12 +16,83 @@ How you should store and interact with data while doing research.
...
+
+<details><summary> Click here  for additional considerations for human data suggested by the NIH. </Summary>
+
+When working with human participant data, including de-identified human data, here are some additional characteristics to look for:
+
+- **Fidelity to Consent**: Uses documented procedures to restrict dataset access and use to those that are consistent with participant consent and changes in consent.
+- **Restricted Use Compliant**: Uses documented procedures to communicate and enforce data use restrictions, such as preventing reidentification or redistribution to unauthorized users.
+- **Privacy**: Implements and provides documentation of measures (for example, tiered access, credentialing of data users, security safeguards against potential breaches) to protect human subjects data from inappropriate access.
+- **Plan for Breach**: Has security measures that include a response plan for detected data breaches.
+- **Download Control*: Controls and audits access to and download of datasets (if download is permitted).
```
Comment (avahoffman) [inline 02-data_management_plan.Rmd:49]:
```suggestion
- **Download Control**: Controls and audits access to and download of datasets (if download is permitted).
```

File: 02-data_management_plan.Rmd [hunk 2]
```diff
@@ -16,12 +16,83 @@ How you should store and interact with data while doing research.
...
+- **Violations**: Has procedures for addressing violations of terms-of-use by users and data mismanagement by the repository.
+- **Request Review**: Makes use of an established and transparent process for reviewing data access requests.
+
+</div>
+
+<div class = "dictionary">
+Data FAIRness - data that is **F**indable, **A**ccessible, **I**nteroperable, and **R**eusable. For more information about data FAIRness, check out  this [manuscript](https://www.nature.com/articles/sdata201618) by @wilkinson_citation_2016.
```
Comment (avahoffman) [inline 02-data_management_plan.Rmd:56]:
```suggestion
**Data FAIRness**

Data that is **F**indable, **A**ccessible, **I**nteroperable, and **R**eusable. For more information about data FAIRness, check out this [manuscript](https://www.nature.com/articles/sdata201618) by @wilkinson_citation_2016.
```

File: 02-data_management_plan.Rmd [hunk 3]
```diff
@@ -16,12 +16,83 @@ How you should store and interact with data while doing research.
...
+    - **Security and Integrity**: Has documented measures in place to meet generally accepted criteria for preventing unauthorized access to, modification of, or release of data, with levels of security that are appropriate to the sensitivity of data.
+    - **Confidentiality**: Has documented capabilities for ensuring that administrative, technical, and physical safeguards are employed to comply with applicable confidentiality, risk management, and continuous monitoring requirements for sensitive data (should your data require such safeguards).
+    - **Common Format**: Allows datasets and metadata downloaded, accessed, or exported from the repository to be in widely used, preferably non-proprietary, formats consistent with those used in the community/communties the repository serves.
+    - **Provenance**: Has mechanisms in place to record the origin, chain of custody, and any modifications to submitted datasets and metadata.
+    - **Retention Policy**: Provides documentation on policies for data retention within the repository.
+
+
+<details><summary> Click here  for additional considerations for human data suggested by the NIH. </Summary>
```
Comment (avahoffman) [inline 02-data_management_plan.Rmd:41]:
```suggestion
<div class = "click_to_expand_block">

<details><summary> Click here  for additional considerations for human data suggested by the NIH. </summary>


```

Comment (avahoffman) [reply 02-data_management_plan.Rmd:41]: Just making it into a block - otherwise might be easy to miss

File: 02-data_management_plan.Rmd [hunk 4]
```diff
@@ -16,12 +16,83 @@ How you should store and interact with data while doing research.
...
+
+- **Fidelity to Consent**: Uses documented procedures to restrict dataset access and use to those that are consistent with participant consent and changes in consent.
+- **Restricted Use Compliant**: Uses documented procedures to communicate and enforce data use restrictions, such as preventing reidentification or redistribution to unauthorized users.
+- **Privacy**: Implements and provides documentation of measures (for example, tiered access, credentialing of data users, security safeguards against potential breaches) to protect human subjects data from inappropriate access.
+- **Plan for Breach**: Has security measures that include a response plan for detected data breaches.
+- **Download Control*: Controls and audits access to and download of datasets (if download is permitted).
+- **Violations**: Has procedures for addressing violations of terms-of-use by users and data mismanagement by the repository.
+- **Request Review**: Makes use of an established and transparent process for reviewing data access requests.
+
+</div>
```
Comment (avahoffman) [inline 02-data_management_plan.Rmd:53]:
```suggestion
</details>

</div>
```

**Review by avahoffman — APPROVED** [review]
Minor comments - looking great!

**Review by ehumph — APPROVED** [review]
Looks good to me - I'm working on a table with Amy about data management that I think will fit in here - will let you know when it's ready (but I think this section is great even without it)

File: 02-data_management_plan.Rmd (hunk 1 above)
Comment (carriewright11) [reply 02-data_management_plan.Rmd:49]: thanks for catching this one :p

File: 02-data_management_plan.Rmd (hunk 3 above)
Comment (carriewright11) [reply 02-data_management_plan.Rmd:41]: yup I like this - that is why I added that css code lol idk why I didn't think to add it here - thanks!

**Comment by carriewright11** [comment]
> Looks good to me - I'm working on a table with Amy about data management that I think will fit in here - will let you know when it's ready (but I think this section is great even without it)

awesome I look forward to the table!

**Pull request merged**